test/Orchard.Tests/project.json

```diff
@@ -15,11 +15,10 @@
     "Orchard.Hosting": "2.0.0-*",
     "Orchard.Parser.Yaml": "2.0.0-*",
     "Orchard.Hosting.Console": "2.0.0-*",
-    "xunit": "2.1.0-*",
-    "xunit.runner.dnx": "2.1.0-*"
+    "xunit.runner.aspnet": "2.0.0-aspnet-*"
   },
   "commands": {
-    "test": "xunit.runner.dnx"
+    "test": "xunit.runner.aspnet"
```

Comment:
There was an issue with the RC1 tools, which meant an error was being thrown when searching for tests which stopped them showing... maybe you are hitting that?